### PR TITLE
Add 2 new rules

### PR DESCRIPTION
**Pull request by jianyexi**

### CODE REVIEW

File: docs/rules/1044.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+### 1044 - XmsLongRunningOperationChanged
+
+**Description**: Checks whether any operation's 'x-ms-long-running-operation' has changed from the previous specification.
+
+**Cause**: This is considered a breaking change.
+
+**Example**: Value of `maxItems` element of property `a` is changed from `5000` to `2000` in the new version.
```
Comment (ruowan) [inline docs/rules/1044.md:7]: This comments is wrong.

Comment (jianyexi) [reply docs/rules/1044.md]: Fixed

File: openapi-diff/src/modeler/AutoRest.Swagger/Model/Schema.cs [hunk 2]
```diff
@@ -270,6 +270,10 @@ private void CompareProperties(ComparisonContext<ServiceDefinition> context, Sch
                             else
                                 context.LogBreakingChange(ComparisonMessages.AddedPropertyInResponse, property.Key);
                         }
+                        if (priorSchema.IsReferenced && property.Value != null)
```
Comment (ruowan) [inline openapi-diff/src/modeler/AutoRest.Swagger/Model/Schema.cs:273]: maybe double output error.

**Review by ruowan — APPROVED** [review]

File: openapi-diff/src/modeler/AutoRest.Swagger/ComparisonMessages.cs [hunk 3]
```diff
@@ -345,5 +353,13 @@ public static class ComparisonMessages
             Message = "The new version has a different '{0}' value than the previous one.",
             Type = MessageType.Update
         };
+
+        public static MessageTemplate LongRunningOperationExtensionChanged = new MessageTemplate
+        {
+            Id = 1036,
```
Comment (PaulMathon) [inline openapi-diff/src/modeler/AutoRest.Swagger/ComparisonMessages.cs:359]: Should be 1044

Comment (jianyexi) [reply openapi-diff/src/modeler/AutoRest.Swagger/ComparisonMessages.cs:359]: BTW, how can you found this issue ?

Comment (PaulMathon) [reply openapi-diff/src/modeler/AutoRest.Swagger/ComparisonMessages.cs:359]: I guess this issue is visible to everyone :)

**Pull request merged**